parsers/ENTSOE.py

```diff
@@ -51,13 +51,13 @@
 ENTSOE_PARAMETER_BY_DESC = {v: k for k, v in ENTSOE_PARAMETER_DESC.items()}
 ENTSOE_PARAMETER_GROUPS = {
     'biomass': ['B01', 'B17'],
-    'coal': ['B02', 'B05', 'B08'],
+    'coal': ['B02', 'B05', 'B07', 'B08'],
```

Comment:
what is ENTSOE B07? This mapping is for all of ENTSOE data - not just EE, so we would have to be careful here.

Reply:
it's "oil shale"

Reply:
Yes, i should check for which country is B07 used ! 👀

Reply:
@alixunderplatz on top of your head dyou remember which other ENTSOE region could use oil shale by any chance?

Reply:
I'm quite sure no other country of entsoe uses oil shale, just Estonia.

Reply:
That will make things easier. Good